program/src/processor.rs

```diff
@@ -353,7 +359,7 @@ impl Processor {
                 receiver_info.clone(),
                 faucet_owner_info.clone(),
             ],
-            &[&[seed.as_ref()]]
+            &[&[seed.as_ref()]]  // What is this seed for?
         )?;
```

Comment:
Can we `invoke_signed` without seed?

Reply:
Yes, you can call `invoke_signed` or just leave the seed slice empty.  Seeds are only required if you are using derived program addresses.

Reply:
Got it, thanks.